src/main/java/org/LogAggregatorTool/operations/LogFilesExecutor.java

```diff
@@ -0,0 +1,31 @@
+package org.LogAggregatorTool.operations;
+
+import org.LogAggregatorTool.utility.Constants;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class LogFilesExecutor {
+    /**
+     * This method creates an executor to create and execute a new thread.
+     * @param pathToLogFilesFolder it is the path to the folder where logfiles are located
+     * @param outputFilePath it is the path, where all the data is to be written
+     * @return this method returns Success/Failure message back to the main thread.
+     */
+    public String executeLogFiles(String pathToLogFilesFolder, String outputFilePath){
+        ExecutorService logFilesExecutor = Executors.newSingleThreadExecutor();
```

Comment:
I suggest to keep the variable name logFilesExecutor as executorService

Reply:
I have changed it.